03week/towersOfHanoi.js

```diff
@@ -19,24 +19,53 @@ function printStacks() {
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
-  // Your code here
-
+//make the 2 tests for this function 
+const movePiece = (valueInArrB4Move, valueInArrB4Drop) => {
+  if(isLegal(valueInArrB4Move[valueInArrB4Move.length-1], valueInArrB4Drop[valueInArrB4Drop.length-1])){
+    const valueOfPickedPiece = valueInArrB4Move.pop();
+    const valueOfArrAfterPieceDop = valueInArrB4Drop.push(valueOfPickedPiece);
+    return valueOfArrAfterPieceDop
+    //returns the length of the new array where you placed the item
+  } 
 }
 
-function isLegal() {
-  // Your code here
-
+// function compares valueInArrB4Move last item value == valueInArrB4Drop last item value
+const isLegal = (valueInArrB4Move, valueInArrB4Drop)=>{
+  if(valueInArrB4Drop === undefined || valueInArrB4Move <= valueInArrB4Drop){
```

Comment:
can be greatly simplified. Return the evaluation and never look for undefined, just look for a falsy value `! valueInArrB4Drop`